Changes.md

```diff
@@ -14,6 +14,7 @@ Improvements
 
 - FrameMask : Improved performance when dealing with long frame ranges.
 - Node : Improved performance of type queries related to dispatch processes.
+- FilterResults : Improved performance of hash using new SceneAlgo::matchingPathsHash
```

Comment:
Let's omit `SceneAlgo::matchingPathsHash()` here, because users don't care how we done it, then we can add a mention of the new SceneAlgo functions in the API section. There's also a missing period.

Reply:
Fixed.